tests/integration/laravel/tests/integration/LaravelTest.php

```diff
@@ -55,7 +55,7 @@ public function testReportsTraceToFile()
 
         $spansByName = $this->groupSpansByName($spans);
 
```

Comment:
add a reference to [Opentelemetry spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#mysql) and these changes are to follow the conventions.